### PR TITLE
feat: Pipeline Server versions of automated self checkout

### PR DESCRIPTION
**Pull request by brian-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

Adding support for DLStreamer pipeline server

## Issue this PR will close

close: #597 

## Test Instructions if applicable

https://github.com/brian-intel/automated-self-checkout/blob/evamPipelines/src/pipeline-server/readme.md

### CODE REVIEW

**Review by antoniomtz — APPROVED** [review]
LGTM

File: src/pipeline-server/docker-compose.pipeline-server.yml [hunk 1]
```diff
@@ -22,34 +22,75 @@ services:
...
       - "${RETAIL_USE_CASE_ROOT:-..}/src/pipeline-server/config.json:/home/pipeline-server/config.json"
 
-  # pipeline-init:
-  #   image: postman/newman
-  #   command: run /postman/pipeline.json --verbose
-  #   volumes:
-  #     - ./postman/:/postman
+  evamclient1:
+    container_name: evam_1
+    image: bmcginn/cv-workshop:2
```
Comment (NeethuES-intel) [inline src/pipeline-server/docker-compose.pipeline-server.yml:51]: do you want your github id in image name or change it to something generic ?

File: src/pipeline-server/Dockerfile.pipeline-server [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (C) 2024 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+FROM intel/dlstreamer-pipeline-server:2022.2.0-ubuntu20-gpu419.40
```
Comment (jim-wang-intel) [inline src/pipeline-server/Dockerfile.pipeline-server:7]: do they have later version? or this is working version for now

File: src/pipelines/yolov5s.sh [hunk 3]
```diff
@@ -17,7 +17,7 @@ fi
...
+gstLaunchCmd="GST_DEBUG=\"GST_TRACER:7\" GST_TRACERS=\"latency_tracer(flags=pipeline,interval=100)\" gst-launch-1.0 $inputsrc ! $DECODE ! gvadetect batch-size=$BATCH_SIZE model-instance-id=odmodel name=detection model=models/object_detection/yolov5s/FP32/yolov5s.xml model-proc=models/object_detection/yolov5s/yolov5s.json threshold=.5 device=$DEVICE $PRE_PROCESS ! $AGGREGATE gvametaconvert name=metaconvert add-empty-results=true ! gvametapublish name=destination file-format=2 file-path=/tmp/results/r$cid\"_gst\".jsonl $OUTPUT 2>&1 | tee >/tmp/results/gst-launch_$cid\"_gst\".log >(stdbuf -oL sed -n -e 's/^.*current: //p' | stdbuf -oL cut -d , -f 1 > /tmp/results/pipeline$cid\"_gst\".log)"
```
Comment (jim-wang-intel) [inline src/pipelines/yolov5s.sh:20]: is FP32 the preferred precision for yolov5? i tried to recall why we used FP16-INT8 version previously...

Comment (brian-intel) [reply src/pipelines/yolov5s.sh:20]: I ran into an issue using FP16 with the pipeline server and changed to FP32. Now that I'm using the latest let me check again with the FP16 model which will have better performance.

File: src/pipeline-server/Dockerfile.pipeline-server (hunk 2 above)
Comment (brian-intel) [reply src/pipeline-server/Dockerfile.pipeline-server:7]: This is the latest public release we have access to

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**